src/components/FeatureLists/PlatformFeatureList.tsx

```diff
@@ -25,8 +27,14 @@ const PlatformFeatureList: React.FC<PlatformFeatureListProps> = ({
             {category.items.map((categoryItem, index) => (
               <FeatureItem
                 linkText={categoryItem.linkText}
-                href={categoryItem.link}
-                isExternal={true}
+                href={
+                  platform === 'javascript' &&
+                  pathname === '/' &&
+                  !categoryItem.link.startsWith('https://ui.docs.amplify.aws/')
+                    ? `${platform}/${categoryItem.link}`
+                    : categoryItem.link
+                }
```

Comment:
```suggestion
                href={
                  platform === 'react' &&
                  pathname === '/' &&
                  !categoryItem.isExternal
                    ? `${platform}/${categoryItem.link}`
                    : categoryItem.link
                }
```

Your PR isn't up to date with next-release/main, but react is the default platform now. We should find a more elegant way to do this so we have less places to hard code the default platform, but I tested locally against latest and this worked for me.

Reply:
Used DEFAULT_PLATFORM so we don't have to hard code here :) thanks

Reply:
NICE 🔥 thank you!